### PR TITLE
Adding tags and updating archetype - Issue #27

### PR DESCRIPTION
**Pull request by Pow3r5**
I went through and added related tags to each of the stories .md files. I also went and updated the archetype to match accordingly and be available for future stories.

### CODE REVIEW

File: content/stories/2021/elana-hashman.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ storyteller: "Elana Hashman"
 storycorps: https://archive.storycorps.org/embed/3519220
 bio: "Elana Hashman is an open source software developer and free software advocate. Contributing to the Python, Debian, Clojure, JavaScript, and Kubernetes communities, she seeks to advance both the ideals and implementation of free and open source software. Elana is a Python Software Foundation Fellow and serves as a member of the Debian Technical Committee. She currently works for Red Hat as a Principal Software Engineer on the OpenShift Container Platform Node Team."
 draft: false
+tags:
+- Python
+- Kubernetes
+- Communitiy
+- Saftey
```
Comment (juliaferraioli) [inline content/stories/2021/elana-hashman.md:13]: This should probably be "Safety" to be safe :-)

File: content/stories/2021/joe-beda.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ storyteller: "Joe Beda"
 storycorps: https://archive.storycorps.org/embed/3455153
 bio: "Joe Beda is a Principal Engineer at VMware. Joe organizes Kubernetes technical direction across VMware and VMware Tanzu. Joe arrived at VMware via the acquisition of Heptio, a leader in the cloud native movement and a company that he co-founded. Previously, at Google, Joe co-created Google Compute Engine and filed the first ever Kubernetes project commit. Joe started his career at Microsoft working on Internet Explorer and Windows. Joe holds a B.S. from Harvey Mudd College and lives in Seattle, Washington with his wife Rachel (a medical doctor and also an HMC alum) and their two children."
 draft: "false"
+tags:
+- Kubernetes
+- Ownership
+- BDFL
+- Communitiy
```
Comment (juliaferraioli) [inline content/stories/2021/joe-beda.md:13]: I might add "Governance" to this list, if you don't mind!

File: content/stories/2021/samson-goddy.md [hunk 3]
```diff
@@ -6,6 +6,11 @@ storyteller: "Samson Goddy"
 storycorps: 'https://archive.storycorps.org/embed/3448105'
 bio: "Samson Goddy is a software engineer, an open source advocate, community builder. He leads Community Relations and Advocacy at Sourcegraph and is the co-founder for Open Source Community Africa; he is a long-time contributor for the Sugar Labs project and serves on Open Source Collective's board. Samson is passionate about building diverse communities around projects and loves teaching."
 draft: false
+tags:
+- Python
+- Diversity
+- Communitiy
+- Africa
```
Comment (juliaferraioli) [inline content/stories/2021/samson-goddy.md:13]: Mind adding "OSCA" here?

File: content/stories/2022/nicolas-steenhout.md [hunk 4]
```diff
@@ -7,6 +7,11 @@ storycorps: 'https://archive.storycorps.org/embed/3526196'
 bio: " Nicolas Steenhout offers real-world insight into accessibility issues. 
 His work began in the mid-'90s. As a developer, he was confronted by accessibility hurdles that were preventing those with disabilities from engaging with a technological revolution. As he began championing web accessibility, Nic transitioned into the non-profit sector where he collaborated with people with a wide variety of impairments. Today, Nic works as an independent consultant for private businesses and non-profits. Having lent his services on three continents, he’s engaged with thousands of individuals with disabilities. He blogs about accessibility at https://incl.ca and he hosts the A11y Rules Podcast."
 draft: false
+tags:
+- Accessibility
+- Web
+- Drupal
+- Diversity
```
Comment (juliaferraioli) [inline content/stories/2022/nicolas-steenhout.md:14]: Similarly, could you add "Disability" and "Inclusion"?

**Review by juliaferraioli — CHANGES_REQUESTED** [review]
It's looking good! Just a few comments about tags that I think could be beneficial.

File: content/stories/2021/joe-beda.md [hunk 5]
```diff
@@ -11,6 +11,7 @@ tags:
 - Ownership
 - BDFL
 - Communitiy
```
Comment (juliaferraioli) [inline content/stories/2021/joe-beda.md:13]: Shoot, I missed that Community is misspelled -- mind fixing that here and in the other files?

**Review by juliaferraioli — CHANGES_REQUESTED** [review]
One last one!

File: content/stories/2021/joe-beda.md (hunk 5 above)
Comment (Pow3r5) [reply content/stories/2021/joe-beda.md:13]: Whoops yep will redo now

**Review by juliaferraioli — APPROVED** [review]
Thank you @Pow3r5! This looks great. 

@all-contributors please add @Pow3r5 for code and content

**Comment by juliaferraioli** [comment]
So appreciate your first contribution, @Pow3r5! Let me know if there are any other great ideas that you have for our site/experience.

**Pull request merged**